src/common/admin_socket.cc

```diff
@@ -693,6 +695,297 @@ class GetdescsHook : public AdminSocketHook {
   }
 };
 
+// Define a macro to simplify adding signals to the map
+#define ADD_SIGNAL(signalName)                 \
+  {                                            \
+    ((const char*)#signalName) + 3, signalName \
+  }
+
+static const std::map<std::string, int> known_signals = {
+  // the following 6 signals are recognized in windows according to
+  // https://learn.microsoft.com/en-us/cpp/c-runtime-library/reference/raise?view=msvc-170
```

Comment:
These come from ANSI C actually.

https://en.wikipedia.org/wiki/C_signal_handling

Reply:
i.e. C89 required these signals so windows apparently must support it.